docs/docs/user/permissions.md

```diff
@@ -26,11 +26,44 @@ Add new topic         | any logged in user
 Remove existing topic | **topic owner**
 Modify topic          | **topic owner**
 
+### Publishing permission
+
+You can configure which services can publish on which topic configuring **topic.auth** section.
+How publisher name is evaluated is deployment specific. It can be extracted from ssl certificate or read from supplied header for instance.
```

Comment:
Can I find in docs information about how to configure publisher name evaluation? Does Hermes provide some kind of default behaviour for this? Imo it is worth to mention about it here